### PR TITLE
WASM vs Docker comparison Link changed

### PR DESCRIPTION
**Pull request by kelvinparmar**
## Explanation

In this PR I resolved the navigation of WASM vs Docker comparison 

## What type of PR is this

Bug

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (hydai) [inline .gitpod.yml:1]: Why do we need this file to resolve the missing link?

**Review by hydai — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (kelvinparmar) [reply .gitpod.yml]: no need to change that file. Removed from the pr

**Review by hydai — APPROVED** [review]

**Comment by hydai** [comment]
Thanks.

**Pull request merged**